kiwi/boot/functions.sh

```diff
@@ -5525,18 +5540,25 @@ function waitForStorageDevice {
     # which could be a whole disk or a partition.
     # the function will wait until the size of the
     # storage device could be obtained or the check
-    # counter equals 4
+    # counter equals 30. For USB devices it waits
+    # only until check counter equals 2.
     # ----
     local IFS=$IFS_ORIG
     local device=$1
     local check=0
+    local transport="`getDeviceTransportType $device`"
+    if [[ $transport == *"usb"* ]];then
```

Comment:
I actually included the regular expression because I could not find a list of possible device transports. So I could not see anywhere the possible values I can get out of it and I though that maybe could be enough if the transport contains usb, however it is true that being more restrictive we are being also safer. So thinking it twice I  think you got a point on that, I prefer just a simple match, better to experience a false negative than a false positive in this case.